src/components/VideoDetailsSidebar/VideoDetailsSidebar.component.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+
+import {
+  VideoList,
+  VideoTitle,
+  Thumbnail,
+  ChannelTitle,
+  StyledVideo,
+} from './VideoDetailsSidebar.styled';
+
+const VideoListItem = ({ data }) => {
```

Comment:
A minor thing you can do to improve this part is to move this component to it's own file, you can keep it in the same folder `src/components/VideoDetailsSidebar` but in different files and just import it here